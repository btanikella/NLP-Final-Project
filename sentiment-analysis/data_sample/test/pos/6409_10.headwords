is
gets
FOREPLAY
Kill-Yourself-Awesome
film
Optimistic
Weird
Manic-depressive
's
IS
see
MUSICAL
WITH
get
had
OH
fun
