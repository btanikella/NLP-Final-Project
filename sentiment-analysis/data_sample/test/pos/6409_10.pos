DT NN VBZ IN DT NN
IN PRP VBZ TO JJ NNS WDT VBP IN NN NN SYM NNP  CC JJR  IN DT NN
NN NN
DT JJ NN VBN IN DT NN CC NN IN DT NNS MD VB RB JJ IN PRP VBP TO DT JJ NNS
PRP VBZ DT JJ NN
JJ
JJ
JJ -LRB- JJ -RRB-
DT VBZ PRP
NNP NNP VBZ NN
RB VB PRP
PRP VBZ DT NN CD
IN NN CD
IN PRP IN DT JJ NN IN DT NN  CC VBG IN NNP  RB PRP$ RB VB DT JJ NN IN IN DT CD
PRP VBD TO VB DT NN IN NNP IN IN CD $
NN CC DT NNS NN
RB PRP VBP CC PRP VBP RB NN  EX IN NN  NN  NNP  CC IN NNP NNP  CC PRP MD RB VB DT NN NN JJR CC FW FW TO DT NNS VBP JJ  CC DT NNS VBP RB NN
