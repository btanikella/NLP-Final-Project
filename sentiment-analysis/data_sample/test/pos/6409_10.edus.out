Sentence #1 (7 tokens):
This film is like a dirge.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=9 CharacterOffsetEnd=13 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=14 CharacterOffsetEnd=16 PartOfSpeech=VBZ] [Text=like CharacterOffsetBegin=17 CharacterOffsetEnd=21 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=22 CharacterOffsetEnd=23 PartOfSpeech=DT] [Text=dirge CharacterOffsetBegin=24 CharacterOffsetEnd=29 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=29 CharacterOffsetEnd=30 PartOfSpeech=.] 
root(ROOT-0, is-3)
det(film-2, This-1)
nsubj(is-3, film-2)
prep(is-3, like-4)
det(dirge-6, a-5)
pobj(like-4, dirge-6)
punct(is-3, .-7)

Sentence #2 (22 tokens):
UNTIL it gets to musical numbers which are like MIND F*CK, but gentler, like a mind caress.
[Text=UNTIL CharacterOffsetBegin=35 CharacterOffsetEnd=40 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=41 CharacterOffsetEnd=43 PartOfSpeech=PRP] [Text=gets CharacterOffsetBegin=44 CharacterOffsetEnd=48 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=49 CharacterOffsetEnd=51 PartOfSpeech=TO] [Text=musical CharacterOffsetBegin=52 CharacterOffsetEnd=59 PartOfSpeech=JJ] [Text=numbers CharacterOffsetBegin=60 CharacterOffsetEnd=67 PartOfSpeech=NNS] [Text=which CharacterOffsetBegin=68 CharacterOffsetEnd=73 PartOfSpeech=WDT] [Text=are CharacterOffsetBegin=74 CharacterOffsetEnd=77 PartOfSpeech=VBP] [Text=like CharacterOffsetBegin=78 CharacterOffsetEnd=82 PartOfSpeech=IN] [Text=MIND CharacterOffsetBegin=83 CharacterOffsetEnd=87 PartOfSpeech=NN] [Text=F CharacterOffsetBegin=88 CharacterOffsetEnd=89 PartOfSpeech=NN] [Text=* CharacterOffsetBegin=89 CharacterOffsetEnd=90 PartOfSpeech=SYM] [Text=CK CharacterOffsetBegin=90 CharacterOffsetEnd=92 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=92 CharacterOffsetEnd=93 PartOfSpeech=,] [Text=but CharacterOffsetBegin=94 CharacterOffsetEnd=97 PartOfSpeech=CC] [Text=gentler CharacterOffsetBegin=98 CharacterOffsetEnd=105 PartOfSpeech=JJR] [Text=, CharacterOffsetBegin=105 CharacterOffsetEnd=106 PartOfSpeech=,] [Text=like CharacterOffsetBegin=107 CharacterOffsetEnd=111 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=112 CharacterOffsetEnd=113 PartOfSpeech=DT] [Text=mind CharacterOffsetBegin=114 CharacterOffsetEnd=118 PartOfSpeech=NN] [Text=caress CharacterOffsetBegin=119 CharacterOffsetEnd=125 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=125 CharacterOffsetEnd=126 PartOfSpeech=.] 
root(ROOT-0, gets-3)
mark(gets-3, UNTIL-1)
nsubj(gets-3, it-2)
prep(gets-3, to-4)
amod(numbers-6, musical-5)
pobj(to-4, numbers-6)
nsubj(are-8, which-7)
rcmod(numbers-6, are-8)
prep(are-8, like-9)
nn(CK-13, MIND-10)
nn(CK-13, F-11)
dep(CK-13, *-12)
pobj(like-9, CK-13)
punct(gets-3, ,-14)
cc(gets-3, but-15)
conj(gets-3, gentler-16)
punct(gentler-16, ,-17)
prep(gentler-16, like-18)
det(mind-20, a-19)
pobj(like-18, mind-20)
dep(mind-20, caress-21)
punct(gets-3, .-22)

Sentence #3 (3 tokens):
MIND FOREPLAY.
[Text=MIND CharacterOffsetBegin=131 CharacterOffsetEnd=135 PartOfSpeech=NN] [Text=FOREPLAY CharacterOffsetBegin=136 CharacterOffsetEnd=144 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=144 CharacterOffsetEnd=145 PartOfSpeech=.] 
root(ROOT-0, FOREPLAY-2)
nn(FOREPLAY-2, MIND-1)
punct(FOREPLAY-2, .-3)

Sentence #4 (24 tokens):
The depressing vibe given from the speed & desperateness of the characters can be pretty Kill-Yourself-Awesome UNTIL you get to the musical numbers.
[Text=The CharacterOffsetBegin=150 CharacterOffsetEnd=153 PartOfSpeech=DT] [Text=depressing CharacterOffsetBegin=154 CharacterOffsetEnd=164 PartOfSpeech=JJ] [Text=vibe CharacterOffsetBegin=165 CharacterOffsetEnd=169 PartOfSpeech=NN] [Text=given CharacterOffsetBegin=170 CharacterOffsetEnd=175 PartOfSpeech=VBN] [Text=from CharacterOffsetBegin=176 CharacterOffsetEnd=180 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=181 CharacterOffsetEnd=184 PartOfSpeech=DT] [Text=speed CharacterOffsetBegin=185 CharacterOffsetEnd=190 PartOfSpeech=NN] [Text=& CharacterOffsetBegin=191 CharacterOffsetEnd=192 PartOfSpeech=CC] [Text=desperateness CharacterOffsetBegin=193 CharacterOffsetEnd=206 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=207 CharacterOffsetEnd=209 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=210 CharacterOffsetEnd=213 PartOfSpeech=DT] [Text=characters CharacterOffsetBegin=214 CharacterOffsetEnd=224 PartOfSpeech=NNS] [Text=can CharacterOffsetBegin=225 CharacterOffsetEnd=228 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=229 CharacterOffsetEnd=231 PartOfSpeech=VB] [Text=pretty CharacterOffsetBegin=232 CharacterOffsetEnd=238 PartOfSpeech=RB] [Text=Kill-Yourself-Awesome CharacterOffsetBegin=239 CharacterOffsetEnd=260 PartOfSpeech=JJ] [Text=UNTIL CharacterOffsetBegin=261 CharacterOffsetEnd=266 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=267 CharacterOffsetEnd=270 PartOfSpeech=PRP] [Text=get CharacterOffsetBegin=271 CharacterOffsetEnd=274 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=275 CharacterOffsetEnd=277 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=278 CharacterOffsetEnd=281 PartOfSpeech=DT] [Text=musical CharacterOffsetBegin=282 CharacterOffsetEnd=289 PartOfSpeech=JJ] [Text=numbers CharacterOffsetBegin=290 CharacterOffsetEnd=297 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=297 CharacterOffsetEnd=298 PartOfSpeech=.] 
root(ROOT-0, Kill-Yourself-Awesome-16)
det(vibe-3, The-1)
amod(vibe-3, depressing-2)
nsubj(Kill-Yourself-Awesome-16, vibe-3)
partmod(vibe-3, given-4)
prep(given-4, from-5)
det(speed-7, the-6)
pobj(from-5, speed-7)
cc(speed-7, &-8)
conj(speed-7, desperateness-9)
prep(speed-7, of-10)
det(characters-12, the-11)
pobj(of-10, characters-12)
aux(Kill-Yourself-Awesome-16, can-13)
cop(Kill-Yourself-Awesome-16, be-14)
advmod(Kill-Yourself-Awesome-16, pretty-15)
mark(get-19, UNTIL-17)
nsubj(get-19, you-18)
ccomp(Kill-Yourself-Awesome-16, get-19)
prep(get-19, to-20)
det(numbers-23, the-21)
amod(numbers-23, musical-22)
pobj(to-20, numbers-23)
punct(Kill-Yourself-Awesome-16, .-24)

Sentence #5 (6 tokens):
It's a great film.
[Text=It CharacterOffsetBegin=303 CharacterOffsetEnd=305 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=305 CharacterOffsetEnd=307 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=308 CharacterOffsetEnd=309 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=310 CharacterOffsetEnd=315 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=316 CharacterOffsetEnd=320 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=320 CharacterOffsetEnd=321 PartOfSpeech=.] 
root(ROOT-0, film-5)
nsubj(film-5, It-1)
cop(film-5, 's-2)
det(film-5, a-3)
amod(film-5, great-4)
punct(film-5, .-6)

Sentence #6 (2 tokens):
Optimistic.
[Text=Optimistic CharacterOffsetBegin=326 CharacterOffsetEnd=336 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=336 CharacterOffsetEnd=337 PartOfSpeech=.] 
root(ROOT-0, Optimistic-1)
punct(Optimistic-1, .-2)

Sentence #7 (2 tokens):
Weird.
[Text=Weird CharacterOffsetBegin=342 CharacterOffsetEnd=347 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=347 CharacterOffsetEnd=348 PartOfSpeech=.] 
root(ROOT-0, Weird-1)
punct(Weird-1, .-2)

Sentence #8 (5 tokens):
Manic-depressive(Bipolar).
[Text=Manic-depressive CharacterOffsetBegin=353 CharacterOffsetEnd=369 PartOfSpeech=JJ] [Text=-LRB- CharacterOffsetBegin=369 CharacterOffsetEnd=370 PartOfSpeech=-LRB-] [Text=Bipolar CharacterOffsetBegin=370 CharacterOffsetEnd=377 PartOfSpeech=JJ] [Text=-RRB- CharacterOffsetBegin=377 CharacterOffsetEnd=378 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=378 CharacterOffsetEnd=379 PartOfSpeech=.] 
root(ROOT-0, Manic-depressive-1)
punct(Bipolar-3, -LRB--2)
dep(Manic-depressive-1, Bipolar-3)
punct(Bipolar-3, -RRB--4)
punct(Manic-depressive-1, .-5)

Sentence #9 (4 tokens):
That's it!
[Text=That CharacterOffsetBegin=384 CharacterOffsetEnd=388 PartOfSpeech=DT] [Text='s CharacterOffsetBegin=388 CharacterOffsetEnd=390 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=391 CharacterOffsetEnd=393 PartOfSpeech=PRP] [Text=! CharacterOffsetBegin=393 CharacterOffsetEnd=394 PartOfSpeech=.] 
root(ROOT-0, 's-2)
nsubj('s-2, That-1)
nsubj('s-2, it-3)
punct('s-2, !-4)

Sentence #10 (5 tokens):
THIS MOVIE IS BIPOLAR.
[Text=THIS CharacterOffsetBegin=399 CharacterOffsetEnd=403 PartOfSpeech=NNP] [Text=MOVIE CharacterOffsetBegin=404 CharacterOffsetEnd=409 PartOfSpeech=NNP] [Text=IS CharacterOffsetBegin=410 CharacterOffsetEnd=412 PartOfSpeech=VBZ] [Text=BIPOLAR CharacterOffsetBegin=413 CharacterOffsetEnd=420 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=420 CharacterOffsetEnd=421 PartOfSpeech=.] 
root(ROOT-0, IS-3)
nn(MOVIE-2, THIS-1)
nsubj(IS-3, MOVIE-2)
dobj(IS-3, BIPOLAR-4)
punct(IS-3, .-5)

Sentence #11 (4 tokens):
anyway see it.
[Text=anyway CharacterOffsetBegin=426 CharacterOffsetEnd=432 PartOfSpeech=RB] [Text=see CharacterOffsetBegin=433 CharacterOffsetEnd=436 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=437 CharacterOffsetEnd=439 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=439 CharacterOffsetEnd=440 PartOfSpeech=.] 
root(ROOT-0, see-2)
advmod(see-2, anyway-1)
dobj(see-2, it-3)
punct(see-2, .-4)

Sentence #12 (5 tokens):
IT'S A MUSICAL!!!
[Text=IT CharacterOffsetBegin=445 CharacterOffsetEnd=447 PartOfSpeech=PRP] [Text='S CharacterOffsetBegin=447 CharacterOffsetEnd=449 PartOfSpeech=VBZ] [Text=A CharacterOffsetBegin=450 CharacterOffsetEnd=451 PartOfSpeech=DT] [Text=MUSICAL CharacterOffsetBegin=452 CharacterOffsetEnd=459 PartOfSpeech=NN] [Text=!!! CharacterOffsetBegin=459 CharacterOffsetEnd=462 PartOfSpeech=CD] 
root(ROOT-0, MUSICAL-4)
nsubj(MUSICAL-4, IT-1)
cop(MUSICAL-4, 'S-2)
det(MUSICAL-4, A-3)
num(MUSICAL-4, !!!-5)

Sentence #13 (3 tokens):
WITH DEPTH!!!!
[Text=WITH CharacterOffsetBegin=467 CharacterOffsetEnd=471 PartOfSpeech=IN] [Text=DEPTH CharacterOffsetBegin=472 CharacterOffsetEnd=477 PartOfSpeech=NN] [Text=!!!! CharacterOffsetBegin=477 CharacterOffsetEnd=481 PartOfSpeech=CD] 
root(ROOT-0, WITH-1)
pobj(WITH-1, DEPTH-2)
num(DEPTH-2, !!!!-3)

Sentence #14 (27 tokens):
If you like the existential dross like The Stranger, or Waiting for Godot, Then your probably get a real kick out of this one.
[Text=If CharacterOffsetBegin=486 CharacterOffsetEnd=488 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=489 CharacterOffsetEnd=492 PartOfSpeech=PRP] [Text=like CharacterOffsetBegin=493 CharacterOffsetEnd=497 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=498 CharacterOffsetEnd=501 PartOfSpeech=DT] [Text=existential CharacterOffsetBegin=502 CharacterOffsetEnd=513 PartOfSpeech=JJ] [Text=dross CharacterOffsetBegin=514 CharacterOffsetEnd=519 PartOfSpeech=NN] [Text=like CharacterOffsetBegin=520 CharacterOffsetEnd=524 PartOfSpeech=IN] [Text=The CharacterOffsetBegin=525 CharacterOffsetEnd=528 PartOfSpeech=DT] [Text=Stranger CharacterOffsetBegin=529 CharacterOffsetEnd=537 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=537 CharacterOffsetEnd=538 PartOfSpeech=,] [Text=or CharacterOffsetBegin=539 CharacterOffsetEnd=541 PartOfSpeech=CC] [Text=Waiting CharacterOffsetBegin=542 CharacterOffsetEnd=549 PartOfSpeech=VBG] [Text=for CharacterOffsetBegin=550 CharacterOffsetEnd=553 PartOfSpeech=IN] [Text=Godot CharacterOffsetBegin=554 CharacterOffsetEnd=559 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=559 CharacterOffsetEnd=560 PartOfSpeech=,] [Text=Then CharacterOffsetBegin=561 CharacterOffsetEnd=565 PartOfSpeech=RB] [Text=your CharacterOffsetBegin=566 CharacterOffsetEnd=570 PartOfSpeech=PRP$] [Text=probably CharacterOffsetBegin=571 CharacterOffsetEnd=579 PartOfSpeech=RB] [Text=get CharacterOffsetBegin=580 CharacterOffsetEnd=583 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=584 CharacterOffsetEnd=585 PartOfSpeech=DT] [Text=real CharacterOffsetBegin=586 CharacterOffsetEnd=590 PartOfSpeech=JJ] [Text=kick CharacterOffsetBegin=591 CharacterOffsetEnd=595 PartOfSpeech=NN] [Text=out CharacterOffsetBegin=596 CharacterOffsetEnd=599 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=600 CharacterOffsetEnd=602 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=603 CharacterOffsetEnd=607 PartOfSpeech=DT] [Text=one CharacterOffsetBegin=608 CharacterOffsetEnd=611 PartOfSpeech=CD] [Text=. CharacterOffsetBegin=611 CharacterOffsetEnd=612 PartOfSpeech=.] 
root(ROOT-0, get-19)
mark(like-3, If-1)
nsubj(like-3, you-2)
advcl(get-19, like-3)
det(dross-6, the-4)
amod(dross-6, existential-5)
pobj(like-3, dross-6)
prep(dross-6, like-7)
det(Stranger-9, The-8)
pobj(like-7, Stranger-9)
punct(Stranger-9, ,-10)
cc(Stranger-9, or-11)
conj(Stranger-9, Waiting-12)
prep(Waiting-12, for-13)
pobj(for-13, Godot-14)
punct(get-19, ,-15)
advmod(get-19, Then-16)
nsubj(get-19, your-17)
advmod(get-19, probably-18)
det(kick-22, a-20)
amod(kick-22, real-21)
dobj(get-19, kick-22)
prep(get-19, out-23)
pcomp(out-23, of-24)
det(one-26, this-25)
pobj(of-24, one-26)
punct(get-19, .-27)

Sentence #15 (13 tokens):
I had to get the DVD through Amazon.com for like 12$.
[Text=I CharacterOffsetBegin=617 CharacterOffsetEnd=618 PartOfSpeech=PRP] [Text=had CharacterOffsetBegin=619 CharacterOffsetEnd=622 PartOfSpeech=VBD] [Text=to CharacterOffsetBegin=623 CharacterOffsetEnd=625 PartOfSpeech=TO] [Text=get CharacterOffsetBegin=626 CharacterOffsetEnd=629 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=630 CharacterOffsetEnd=633 PartOfSpeech=DT] [Text=DVD CharacterOffsetBegin=634 CharacterOffsetEnd=637 PartOfSpeech=NN] [Text=through CharacterOffsetBegin=638 CharacterOffsetEnd=645 PartOfSpeech=IN] [Text=Amazon.com CharacterOffsetBegin=646 CharacterOffsetEnd=656 PartOfSpeech=NNP] [Text=for CharacterOffsetBegin=657 CharacterOffsetEnd=660 PartOfSpeech=IN] [Text=like CharacterOffsetBegin=661 CharacterOffsetEnd=665 PartOfSpeech=IN] [Text=12 CharacterOffsetBegin=666 CharacterOffsetEnd=668 PartOfSpeech=CD] [Text=$ CharacterOffsetBegin=668 CharacterOffsetEnd=669 PartOfSpeech=$] [Text=. CharacterOffsetBegin=669 CharacterOffsetEnd=670 PartOfSpeech=.] 
root(ROOT-0, had-2)
nsubj(had-2, I-1)
aux(get-4, to-3)
xcomp(had-2, get-4)
det(DVD-6, the-5)
dobj(get-4, DVD-6)
prep(get-4, through-7)
pobj(through-7, Amazon.com-8)
prep(get-4, for-9)
pcomp(for-9, like-10)
num($-12, 12-11)
pobj(like-10, $-12)
punct(had-2, .-13)

Sentence #16 (6 tokens):
OH & the songs rock.
[Text=OH CharacterOffsetBegin=675 CharacterOffsetEnd=677 PartOfSpeech=NN] [Text=& CharacterOffsetBegin=678 CharacterOffsetEnd=679 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=680 CharacterOffsetEnd=683 PartOfSpeech=DT] [Text=songs CharacterOffsetBegin=684 CharacterOffsetEnd=689 PartOfSpeech=NNS] [Text=rock CharacterOffsetBegin=690 CharacterOffsetEnd=694 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=694 CharacterOffsetEnd=695 PartOfSpeech=.] 
root(ROOT-0, OH-1)
cc(OH-1, &-2)
det(rock-5, the-3)
nn(rock-5, songs-4)
conj(OH-1, rock-5)
punct(OH-1, .-6)

Sentence #17 (47 tokens):
well they rock but they aren't rock, there like calypso, jazz, Broadway, but by Grace Chung, & I can't find the soundtrack NOWHERE< but i wanna the songs are great, & the dances are so fun.
[Text=well CharacterOffsetBegin=700 CharacterOffsetEnd=704 PartOfSpeech=RB] [Text=they CharacterOffsetBegin=705 CharacterOffsetEnd=709 PartOfSpeech=PRP] [Text=rock CharacterOffsetBegin=710 CharacterOffsetEnd=714 PartOfSpeech=VBP] [Text=but CharacterOffsetBegin=715 CharacterOffsetEnd=718 PartOfSpeech=CC] [Text=they CharacterOffsetBegin=719 CharacterOffsetEnd=723 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=724 CharacterOffsetEnd=727 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=727 CharacterOffsetEnd=730 PartOfSpeech=RB] [Text=rock CharacterOffsetBegin=731 CharacterOffsetEnd=735 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=735 CharacterOffsetEnd=736 PartOfSpeech=,] [Text=there CharacterOffsetBegin=737 CharacterOffsetEnd=742 PartOfSpeech=EX] [Text=like CharacterOffsetBegin=743 CharacterOffsetEnd=747 PartOfSpeech=IN] [Text=calypso CharacterOffsetBegin=748 CharacterOffsetEnd=755 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=755 CharacterOffsetEnd=756 PartOfSpeech=,] [Text=jazz CharacterOffsetBegin=757 CharacterOffsetEnd=761 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=761 CharacterOffsetEnd=762 PartOfSpeech=,] [Text=Broadway CharacterOffsetBegin=763 CharacterOffsetEnd=771 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=771 CharacterOffsetEnd=772 PartOfSpeech=,] [Text=but CharacterOffsetBegin=773 CharacterOffsetEnd=776 PartOfSpeech=CC] [Text=by CharacterOffsetBegin=777 CharacterOffsetEnd=779 PartOfSpeech=IN] [Text=Grace CharacterOffsetBegin=780 CharacterOffsetEnd=785 PartOfSpeech=NNP] [Text=Chung CharacterOffsetBegin=786 CharacterOffsetEnd=791 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=791 CharacterOffsetEnd=792 PartOfSpeech=,] [Text=& CharacterOffsetBegin=793 CharacterOffsetEnd=794 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=795 CharacterOffsetEnd=796 PartOfSpeech=PRP] [Text=ca CharacterOffsetBegin=797 CharacterOffsetEnd=799 PartOfSpeech=MD] [Text=n't CharacterOffsetBegin=799 CharacterOffsetEnd=802 PartOfSpeech=RB] [Text=find CharacterOffsetBegin=803 CharacterOffsetEnd=807 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=808 CharacterOffsetEnd=811 PartOfSpeech=DT] [Text=soundtrack CharacterOffsetBegin=812 CharacterOffsetEnd=822 PartOfSpeech=NN] [Text=NOWHERE CharacterOffsetBegin=823 CharacterOffsetEnd=830 PartOfSpeech=NN] [Text=< CharacterOffsetBegin=830 CharacterOffsetEnd=831 PartOfSpeech=JJR] [Text=but CharacterOffsetBegin=832 CharacterOffsetEnd=835 PartOfSpeech=CC] [Text=i CharacterOffsetBegin=836 CharacterOffsetEnd=837 PartOfSpeech=FW] [Text=wan CharacterOffsetBegin=838 CharacterOffsetEnd=841 PartOfSpeech=FW] [Text=na CharacterOffsetBegin=841 CharacterOffsetEnd=843 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=844 CharacterOffsetEnd=847 PartOfSpeech=DT] [Text=songs CharacterOffsetBegin=848 CharacterOffsetEnd=853 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=854 CharacterOffsetEnd=857 PartOfSpeech=VBP] [Text=great CharacterOffsetBegin=858 CharacterOffsetEnd=863 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=863 CharacterOffsetEnd=864 PartOfSpeech=,] [Text=& CharacterOffsetBegin=865 CharacterOffsetEnd=866 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=867 CharacterOffsetEnd=870 PartOfSpeech=DT] [Text=dances CharacterOffsetBegin=871 CharacterOffsetEnd=877 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=878 CharacterOffsetEnd=881 PartOfSpeech=VBP] [Text=so CharacterOffsetBegin=882 CharacterOffsetEnd=884 PartOfSpeech=RB] [Text=fun CharacterOffsetBegin=885 CharacterOffsetEnd=888 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=888 CharacterOffsetEnd=889 PartOfSpeech=.] 
root(ROOT-0, fun-46)
advmod(rock-3, well-1)
nsubj(rock-3, they-2)
nsubj(fun-46, rock-3)
cc(rock-3, but-4)
nsubj(rock-8, they-5)
cop(rock-8, are-6)
neg(rock-8, n't-7)
conj(rock-3, rock-8)
punct(rock-3, ,-9)
expl(find-27, there-10)
mark(find-27, like-11)
pobj(like-11, calypso-12)
punct(calypso-12, ,-13)
conj(calypso-12, jazz-14)
punct(calypso-12, ,-15)
appos(calypso-12, Broadway-16)
punct(calypso-12, ,-17)
cc(like-11, but-18)
conj(like-11, by-19)
nn(Chung-21, Grace-20)
pobj(by-19, Chung-21)
punct(like-11, ,-22)
cc(like-11, &-23)
conj(like-11, I-24)
aux(find-27, ca-25)
neg(find-27, n't-26)
conj(rock-3, find-27)
det(soundtrack-29, the-28)
dobj(find-27, soundtrack-29)
nn(<-31, NOWHERE-30)
dep(soundtrack-29, <-31)
cc(<-31, but-32)
dep(great-39, i-33)
dep(great-39, wan-34)
dep(great-39, na-35)
det(songs-37, the-36)
nsubj(great-39, songs-37)
cop(great-39, are-38)
conj(<-31, great-39)
punct(<-31, ,-40)
cc(<-31, &-41)
det(dances-43, the-42)
conj(<-31, dances-43)
cop(fun-46, are-44)
advmod(fun-46, so-45)
punct(fun-46, .-47)

